src/module/data/pseudo-documents/advancements/trait-advancement.mjs

```diff
@@ -112,14 +112,16 @@ export default class TraitAdvancement extends BaseAdvancement {
       const submit = dialog.element.querySelector(".form-footer [type=submit]");
       multiCheckbox.addEventListener("change", () => {
         for (const checkbox of multiCheckbox.querySelectorAll("input")) checkbox.disabled = !multiCheckbox.value.includes(checkbox.value) && (multiCheckbox.value.length >= chooseN);
-        submit.disabled = multiCheckbox.value.length !== chooseN;
+        // You don't have to pick all languages when they are granted
+        if (this.type !== "language") submit.disabled = multiCheckbox.value.length !== chooseN;
+        else submit.disabled = multiCheckbox.value.length > chooseN;
```

Comment:
I think it's fine to take the same approach for skills, too. Users can pick 1 of N and leave the rest for later.

```suggestion
        submit.disabled = multiCheckbox.value.length > chooseN;
```